src/components/Common/ConfirmDialog.tsx

```diff
@@ -71,15 +71,17 @@ export const ConfirmDialog = ({
                 )}
               </div>
               <div className="flex items-center gap-3">
-                <button
-                  className="button button-secondary"
-                  onClick={() => {
-                    onClose(false);
-                  }}
-                  data-qa="cancel-dialog"
-                >
-                  {cancelLabel}
-                </button>
+                {cancelLabel && (
+                  <button
+                    className="button button-secondary"
+                    onClick={() => {
+                      onClose(false);
+                    }}
+                    data-qa="cancel-dialog"
+                  >
+                    {cancelLabel}
+                  </button>
+                )}
                 <button
                   ref={confirmLabelRef}
```

Comment:
set `autoFocus` to avoid tooltip displaying when click `Cancel`
![image](https://github.com/epam/ai-dial-chat/assets/20163971/66e90486-5c0b-449e-8ce0-47d851da955c)